tempest-internal/build.gradle.kts

```diff
@@ -1,6 +1,11 @@
+import com.vanniktech.maven.publish.JavadocJar.Dokka
+import com.vanniktech.maven.publish.MavenPublishBaseExtension
+import com.vanniktech.maven.publish.KotlinMultiplatform
+
 plugins {
   kotlin("jvm")
   `java-library`
```

Comment:
Delete? (you’ve deleted `java-library` elsewhere; seems unnecessary)

Reply:
The other deletions where a find/replace mistake. According to [gradle docs](https://docs.gradle.org/current/userguide/java_library_plugin.html) this is necessary for the `api/implementation` functions.